lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -224,6 +224,16 @@ def html_link(paths, full_path: true)
       paths.first(paths.count - 1).join(", ") + " & " + paths.last
     end
 
+    # @!group Github Misc
+    # Tell github to ingore any inline message whihc is not in diff's range, not post as main comment.
```

Comment:
Capital "H" in GitHub please, for "Github Misc"

Nice work on the docs! What about a small evolution: "Use to ignore inline messages which lay outside a diff's range, thereby not posting them in the main comment." instead?